Javascript/assignment4/promises.js

```diff
@@ -0,0 +1,34 @@
+const task1 =
+    new Promise(function (resolve, reject) {
+        setTimeout(function () {
+            resolve('Hello');
+        }, 1000)
+    });
+
+const task2 =
+    new Promise(function (resolve, reject) {
+        setTimeout(function () {
+            resolve('Hello World');
+        }, 1000)
+    });
+
+const task3 =
+    new Promise(function (resolve, reject) {
+        setTimeout(function () {
+            resolve('Hello World Hello World');
+        }, 1000)
+    });
+
+    let promise = async () => {
+        for (let promise of [task1, task2,task3]) {
```

Comment:
Need a solution without using the for of loop.  for of loop was created for iterators..